q2_longitudinal/plugin_setup.py

```diff
@@ -172,13 +172,13 @@
         'Performs paired difference testing between samples from each '
         'subject. Sample pairs may represent a typical intervention study '
         '(e.g., samples collected pre- and post-treatment), paired samples '
-        'from two different timepoints (e.g., in a longitudinal study design),'
-        ' or identical samples receiving different treatments. This action '
-        'tests whether the change in a numeric metadata value "metric" differs'
-        ' from zero and differs between groups (e.g., groups of subjects '
-        'receiving different treatments), and produces boxplots of paired '
-        'difference distributions for each group. Note that "metric" can be '
-        'derived from a feature table or metadata.')
+        'from two different timepoints (e.g., in a longitudinal study design)'
+        ', or identical samples receiving different treatments. This action '
```

Comment:
sorry getting pedantic (or canonical?) now: please move "design)" down to this line too, it will be easier to read (starting the line with a comma is sort of visually jarring)